File: WorkoutTracker.Domain/Users/UserMinMaxReps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WorkoutTracker.Domain.BaseClasses;

namespace WorkoutTracker.Domain.Users
{
    public class UserMinMaxReps : Entity
    {
        public int UserSettingsId { get; set; }
        public UserGoal Goal { get; set; }
        public SetType SetType { get; set; }
        public ushort? Duration { get; set; }
        public byte MinReps { get; set; }
        public byte MaxReps { get; set; }
    }
}

File: workout-tracker/src/app/core/models/user-min-max-reps.ts
import { Entity } from "app/shared/models/entity";
import { SetType } from "app/workouts/enums/set-type";

export class UserMinMaxReps extends Entity {
  public userSettingsId: number = 0;
  public setType: SetType = SetType.Repetition;
  public duration: number | null = null;
  public minReps: number = 0;
  public maxReps: number = 0;
}
